### ISSUE TITLE
Chart and title not displaying in page

### ISSUE THREAD

**Issue by seanbode** [krispo/angular-nvd3#668]
I am trying to display metric data of how many times a user hits play of a song in my music player application displayed on a pie chart using the nvd3 library. I am not getting errors in the console. 
this is my controller `MetricsCtrl.js`:
```
angular
  .module('blocJams')
  .controller('MetricsCtrl', function ($scope, $rootScope, Metrics) {
    $scope.metric = { chart: Metrics.chart }
    $scope.buttonCallback = function() {
      alert("button clicked");
      Metrics.listSongsPlayed()
    };
    // $scope.data = [
    //   {song: "blue", plays: 5 }
    // ]
  });
```
this is my service `Metrics.js`:
```
(function() {
  function Metrics($rootScope, $window) {
    if (!$window.localStorage.getItem("songPlays")) {
      $window.localStorage.songPlays = JSON.stringify([]);
    };

    return {
      // Function that records a metric object by pushing it to the $rootScope array
      registerSongPlay: function(songObj) {
        // Add time to event register// moment.js
        if (songObj != undefined) {
          songObj['playedAt'] = new Date()
          var plays = JSON.parse($window.localStorage.songPlays)
          plays.push(songObj);
          $window.localStorage.songPlays = JSON.stringify(plays)
        };
      },
      chart: {
        type: "pieChart",
        height: 500,
        x: function (d){return d.song;},
        y: function (d){return d.plays;},
        showLabels: true,
        duration: 500,
        labelThreshold: 0.01,
        labelSunbeamLayout: true,
        legend: {
          margin: {
            top: 5,
            right: 35,
            bottom: 5,
            left: 0
          }
        }
      },
      title: {
        enable: true,
        text: "Songs Played: Service rendered",
        className: "h4",
        css:
        {
          width: 0,
          textAlign: "center"
        }
      },
      listSongsPlayed: function() {
        var songs = [];
        angular.forEach(JSON.parse($window.localStorage.songPlays), function(song) {
          songs.push(song.title);
        });
        return songs;
      }
    }
  }
  angular.module ('blocJams').service('Metrics', Metrics);
})();
```

this is my html `metrics.html`:
```
<nvd3 options="metric" data="data">
</nvd3>
<div>
</div>
<h1>HTML!</h1>
```
and this is my main page with the ui router in it `index.html`:
````
<!DOCTYPE html>
<html ng-app="blocJams">
  <head lang="en">
    <meta charset="UTF-8">
    <title>Bloc Jams</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Open+Sans:400,800,600,700,300">
    <link rel="stylesheet" type="text/css" href="http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
    <link rel="stylesheet" type="text/css" href="styles/normalize.css">
    <link rel="stylesheet" type="text/css" href="styles/main.css">
    <link rel="stylesheet" type="text/css" href="styles/landing.css">
    <link rel="stylesheet" type="text/css" href="/styles/collection.css">
    <link rel="stylesheet" type="text/css" href="/styles/album.css">
    <link rel="stylesheet" type="text/css" href="/styles/player_bar.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.8.1/nv.d3.min.css"/>
  </head>

  <body class="landing">
    <ui-view></ui-view>
    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.8.1/nv.d3.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-nvd3/1.0.9/angular-nvd3.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/buzz/1.1.10/buzz.min.js"></script>
    <script src="/scripts/app.js"></script>
    <script src="/scripts/services/Fixtures.js"></script>
    <script src="/scripts/services/SongPlayer.js"></script>
    <script src="/scripts/services/Metrics.js"></script>
    <script src="/scripts/controllers/LandingCtrl.js"></script>
    <script src="/scripts/controllers/CollectionCtrl.js"></script>
    <script src="/scripts/controllers/AlbumCtrl.js"></script>
    <script src="/scripts/controllers/PlayerBarCtrl.js"></script>
    <script src="/scripts/controllers/MetricsCtrl.js"></script>
    <script src="/scripts/directives/seekBar.js"></script>
    <script src="/scripts/filters/timecode.js"></script>
    <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
  </body>
</html>
```
`

**Comment by iiadh** [comment]
hello i encountred this problem, i created the options into a service but the chart doesnt displayed , ifu resolved it, can you tell me how plzz
